app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -322,27 +343,25 @@ def parse_stack(deployment)
           :parameters     => stack_parameters(deployment),
           :resource_group => deployment.resource_group,
 
-          :orchestration_template => stack_template(deployment)
+          :orchestration_template => raw_stack_template(deployment)
         }
         return uid, new_result
       end
 
-      def stack_template(deployment)
+      def raw_stack_template(deployment)
```

Comment:
Couldn't put my finger on this one. Thanks for putting it into words.
Yes, please change from raw.
I had the expectation / confusion